assets/section-main-product.css

```diff
@@ -1148,6 +1148,46 @@ a.product__text {
   border-color: rgb(var(--color-foreground));
 }
 
+.image-magnify-full-size {
+  cursor: zoom-out;
+  z-index: 1;
+  margin: 0;
+  border-radius: calc(var(--media-radius) - var(--media-border-width));
+}
+
+.image-magnify-hover {
+  cursor: zoom-in;
+}
+
+.product__modal-opener--image .product__media-zoom-none,
+.product__media-icon--none {
+  display: none;
+}
+
+@media (hover: hover) {
+  .product__media-zoom-hover {
+    display: none;
+  }
+
+  .product__media-icon--hover {
+    display: none;
+  }
+}
+
+@media screen and (max-width: 23.4375rem) {
```

Comment:
Why was this the value of the `max-width` here ?

Reply:
Hmm we're usually relying on 2 sizes, `750px` (or `749` when using `max-width`) and `990px` (or `989` when using `max-width`).
I'd default to `px` values for media queries, as `rem` values will be different based on the `font size scale` picked ([screenshot](https://screenshot.click/24-04-itc5a-p9m1z.png)). 
Do you forsee an issue with switching to `749px` instead ?